<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeChooser</class>
 <widget class="QDialog" name="ModeChooser">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>153</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>153</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>153</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Log in</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="nicknameLayout">
     <item>
      <widget class="QLabel" name="nicknameLabel">
       <property name="text">
        <string>Nick:</string>
       </property>
       <property name="buddy">
        <cstring>nicknameInput</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nicknameInput">
       <property name="maxLength">
        <number>16</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="addressLayout">
     <item>
      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <widget class="QLabel" name="addressLabel">
         <property name="text">
          <string>Address:</string>
         </property>
         <property name="buddy">
          <cstring>addressInput</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="addressInput"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rightLayout">
       <item>
        <widget class="QLabel" name="portLabel">
         <property name="text">
          <string>Port:</string>
         </property>
         <property name="buddy">
          <cstring>portInput</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="portInput">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="serverCheckbox">
     <property name="text">
      <string>Create server</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="continueButton">
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nicknameInput</tabstop>
  <tabstop>addressInput</tabstop>
  <tabstop>portInput</tabstop>
  <tabstop>serverCheckbox</tabstop>
  <tabstop>continueButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>serverCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>addressInput</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serverCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>addressLabel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
